spec/services/saml_endpoint_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 
 describe SamlEndpoint do
-  let(:path) { '/api/saml/auth2022' }
+  let(:path) { '/api/saml/auth2023' }
```

Comment:
Wasn't sure about these since they were 2 years prior, thank you for confirming!